recipe-box-backend/src/main/java/com/endava/recipebox/service/impl/RecipeServiceImpl.java

```diff
@@ -142,10 +142,10 @@ public RecipeDetailsDTO getDetailedRecipeById(Long recipeId, Long userId) {
     @Transactional
     public String updateRecipe(RecipeEditRequestDTO recipeAddRequestDTO, Long userId){
         Recipe recipe = recipeRepository.findById(recipeAddRequestDTO.getId())
-                .orElseThrow(() -> new EntityNotFoundException("Recipe not found"));
+                .orElseThrow(() -> new BadRequestException("Recipe not found."));
 
         User user = userRepository.findById(userId)
-                .orElseThrow(() -> new EntityNotFoundException("User not found"));
+                .orElseThrow(() -> new BadRequestException("User not found."));
```

Comment:
Why did you change the exception type?

Reply:
Bad request is mapped to 400, any http status with 4xx - represents an error produced by caller

Reply:
I changed the exception because that EntityNotFoundException is not a custom exception and I get Internal Server error 500 in postman, with my exception I am informed corectly about the error